templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -437,19 +463,12 @@ contents:
       ovs-vsctl --timeout=30 --if-exists del-br br0
       exit 0
     elif [ "$1" == "OpenShiftSDN" ]; then
-      # Revert changes made by /usr/local/bin/configure-ovs.sh.
-      update_nm_conn_files br-ex phys0
-      rm_nm_conn_files
+      # Revert changes made by /usr/local/bin/configure-ovs.sh during SDN migration.
+      remove_ovn_bridges br-ex phys0
       if [ -d "/sys/class/net/br-ex1" ]; then
-        update_nm_conn_files br-ex1 phys1
-        rm_nm_conn_files
+        remove_ovn_bridges br-ex1 phys1
       fi
 
```

Comment:
I would add here the removal of bridges created by ovn-kubernetes:
```
# remove bridges created by ovn-kuberenetes
ovs-vsctl --timeout=30 --if-exists del-br br-int -- --if-exists del-br br-local
```

Reply:
fixed.